rpc/grpc/types.proto

```diff
@@ -1,7 +1,7 @@
 syntax = "proto3";
 package core_grpc;
 
-import "github.com/tendermint/abci/types/types.proto";
+import "github.com/tendermint/abci/blob/master/types/types.proto";
```

Comment:
really? was this broken ?

Reply:
I did not change this

Reply:
@zramsay can you explain this?

Reply:
https://github.com/tendermint/tendermint/pull/686